src/main_macro.rs

```diff
@@ -29,23 +31,68 @@
 ///     println!("{}", x);
 /// });
 /// ```
+///
+/// With command line arguments.
+///
+/// ```rust,ignore
```

Comment:
why ignore and not no_run?

Reply:
Let's use the other comment to discuss this :)